src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -269,20 +276,35 @@ public DirectoryInfo ComputeProjectBaseDir(IList<DirectoryInfo> projectPaths)
             else if (PathHelper.BestCommonPrefix(projectPaths, pathComparer) is { } commonPrefix)
             {
                 logger.LogDebug(Resources.MSG_UsingLongestCommonBaseDir, commonPrefix.FullName, Environment.NewLine + string.Join($"{Environment.NewLine}", projectPaths.Select(x => x.FullName)));
-                foreach (var projectOutsideCommonPrefix in projectPaths.Where(x => !x.FullName.StartsWith(commonPrefix.FullName, pathComparison)))
```

Comment:
On line 271, we discussed about changing `.All` with `.Any`.
How come this was not implemented?

Reply:
I discussed with Denis and decided to go with the common root, eliminating the generated projects. I'm not sure anymore about using `.Any`.